communication/zoom-guidelines.md

```diff
@@ -0,0 +1,39 @@
+# Zoom Guidelines
+
+Zoom is the main video communication platform for Kubernetes outside of our mailing lists. 
+It is used for running the [community meeting](https://github.com/kubernetes/community/blob/master/events/community-meeting.md) and SIG meetings. 
+Since the Zoom meetings are open to the general public, a Zoom host might have to moderate a meeting if a person is in violation of the code of conduct. 
+
+These guidelines are meant as a tool to help Kubernetes members manage their Zoom resources. 
+Check the main [moderation](./moderation.md) page for more information on other tools and general moderation guidelines.
+
+## Code of Conduct
+Kubernetes adheres to Cloud Native Compute Foundation's [Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md) throughout the project, and includes all communication mediums.
+
+## Moderation
+
+Zoom has documentation on how to use their moderation tools: 
+
+- https://support.zoom.us/hc/en-us/articles/201362603-Host-Controls-in-a-Meeting
+
+Check the "Screen Share Controls" (via the ^ next to Share Screen): Select who can share in your meeting and if you want only the host or any participant to be able to start a new share when someone is sharing.  
```

Comment:
should we have them do this by default instead of optional at this point?

Reply:
yes